api/src/main/java/com/codeforcommunity/dto/emailer/EmailAttachment.java

```diff
@@ -0,0 +1,105 @@
+package com.codeforcommunity.dto.emailer;
+
+import com.codeforcommunity.dto.ApiDto;
+import com.codeforcommunity.exceptions.BadRequestImageException;
+import com.codeforcommunity.exceptions.HandledException;
+import com.codeforcommunity.exceptions.MalformedParameterException;
+
+import org.simplejavamail.api.email.AttachmentResource;
+
+import javax.mail.util.ByteArrayDataSource;
+
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+
+public class EmailAttachment extends ApiDto {
+    private String name;
+    private String data;
+
+    public EmailAttachment(String name, String data) {
+        this.name = name;
+        this.data = data;
+    }
+
+    private EmailAttachment() {}
+
+    private static String getFileMimeType(String base64File) {
+        // Expected Base64 format: "data:image/{extension};base64,{imageData}"
```

Comment:
Same comment as on reject email - avoid using comments and instead focus on good names (which it looks like you have, so you can delete there)